### ISSUE TITLE
240p (320x240)

### ISSUE THREAD

**Issue by Redemp** [Siddy212/canvas-es#8]
Would you consider adding in support for 4:3 @ 320x240 resolution in the future.
Would love to be able to use this theme on a CRT in a progressive resolution.

Best Regards 
/Rion

See our work here on Batocera for CRT output.
[Batocera-CRT-Script](https://github.com/ZFEbHVUE/Batocera-CRT-Script)

**Comment by Mirado-cloud** [comment]
Would love this, 100%.

**Comment by whatnowmax** [comment]
Many architectures cannot do interlaced resolutions (Intel in particular) which makes the beautiful 480i themes we have access to, inaccessible. 240p has half of the vertical pixels available which only really matters for reading text. Unfortunately reading text is 90% of the purpose of a ES theme.

There is also the factor of accessibility. Interlaced resolutions give me an incredible headache on most crts. I'm sure I'm not alone. 

240p theme would also be the "lowest common denominator" even for lcd displays that are low resolution or very small. The benefit would extend past crt users.

**Comment by abreucatarina** [comment]
I also agree. Most themes were clearly built for LCDs, and feel weird when displayed on CRTs, at least for me.

**Comment by rootCompute** [comment]
Would be great if this could be made possible!
